src/__snapshots__/storyshots.test.js.snap

```diff
@@ -5,13 +5,13 @@ exports[`Storyshots App Alert Text 1`] = `
   class="storybook-snapshot-container"
```

Comment:
My guess is that you might have some unrelated changes locally which aren't part of this commit. I would just revert these changes-- nothing in this PR should change the storyshots.